### ISSUE TITLE
`GetAllModelList` endpoint not working !

### ISSUE THREAD

**Issue by zaza-dahmane** [Cdiscount/API-MarketPlace-SDK-PHP#39]
I'm currently trying to get model list for all the categories, but this call is timing out.
in [cdiscount website](https://dev.cdiscount.com/marketplace/) I've found this: 

> 

Timeout on GetAllModelList method | In progress | Not defined | use the GetModelList method
-- | -- | -- | --

Do you know any other way to get all categories model list ?

**Comment by yoandm** [comment]
I saw it to. I need all the models too. Do you have any news about it ?

**Comment by gregz1** [comment]
hello i just tried the getModelList Method and it seems to work well.
Did you provide a category code as parameter ?
you can test this method here : https://cdscntmkpapiwebapp.herokuapp.com/ProductManager/GetModelListRequest

**Comment by yoandm** [comment]
Yes ok for GetModelList but not for GetAllModelList (to get all the models).
Thank you.

**Comment by gregz1** [comment]
at the moment there is only one model available on Cdiscount Marketplace. it is returned several time in the GetAllModelList this the reason why it is systematically in timeout.
At the moment if you need to have every Models you should call GetModelList for each category code provided by GetAllowedCategoryTree.
In addition GetModelList provide additional informations according to the category provided.

**Comment by yoandm** [comment]
Only one model ? The same for all categories ?
So we don't need to get every Model if there is only one ?
I want to be sure to understand. Mandatories fields are not the same ?
Thank you Greg.
